analyzer/src/main/resources/gme.gradle

```diff
@@ -0,0 +1,22 @@
+/**
+ * This file is injected by the Gradle Manipulation Tool during the analysis phase
+ */
+
+buildscript {
+    repositories {
+        mavenCentral()
+        // Maven local isn't required for production but its useful during development.
+        mavenLocal()
+    }
+
+    dependencies {
+        classpath "org.jboss.gm.manipulation:manipulation:${project.version}"
+    }
+}
+
+// If we are not running the analyse plugin then apply the manipulation plugin.
+// This avoids issues with rerunning in local environments.
+if (!project.hasProperty("gmeAnalyse")) {
```

Comment:
Where does that come into play? I don't see any reference to this property anywhere (though I might just have missed it)…

Reply:
Its linked to my init.gradle which sets the value to true. 
I'll create a separate ticket for this as I think we should store the init.gradle in the repo somewhere so we can discuss and have things like
```
allprojects {
    ext.gmeAnalyse = true
    apply plugin: org.jboss.gm.analyzer.alignment.AlignmentPlugin
```

Reply:
I agree about storing the `init.gradle` in the repo: I had actually started working on it at some point.